### ISSUE TITLE
launching ppserver.py without port fails

### ISSUE THREAD

**Issue by SKKx** [uqfoundation/ppft#3]
Launch in a shell the following command :
```
$ ppserver.py -a
```
Now try to launch sum_primes.py example with '127.0.0.1' in ppservers tuple.
```
$ python sum_primes.py 0
```
It just freeze while with the original pp version, it works.
In order to get ppft to work with sum_primes.py example you have to specify which port to use and listen on.
```
$ ppserver.py -a -p60000
```
and '127.0.0.1:60000' in ppservers tuple.

**Comment by mmckerns** [comment]
Yep, I can replicate that.  I think there's an issue with `auto detect` somewhere.

**Comment by mmckerns** [comment]
If you don't use the `-a` flag, you get exactly the same behavior.  So essentially, it appears that `-a` is ignored.

**Comment by mmckerns** [comment]
Also, you could provide which version of python and which OS, in case that matters?

**Comment by SKKx** [comment]
I use python 2.7.2 on OSX 10.8.2 desktop machine, and python 2.7.9 on Ubuntus 15.04 distant servers.
If I remove the -a flag from the servers, then the whole process fail (silently). My OSX pp process do not find the distant servers and I don't know why as no error message appears. Everything looks like working fine, but nothing happens. I fixed this issue by adding the port number.

That said, I agree that if we remove the -a flag in a localhost test environment, it works.

**Comment by mmckerns** [comment]
Ok, thanks.  I know when I converted ppft from 2 to 2 and 3, I introduced one bug.  This may be a second.  I'll look for it when I can, as it's somewhat lower priority.
